syspage.c

```diff
@@ -553,6 +566,20 @@ int syspage_getMapTop(const char *map, addr_t *addr)
 }
 
 
+const char *syspage_getMapName(u32 id)
+{
+	plo_map_t *pmap = syspage_common.maps;
+	int i;
+
+	for (i = 0; i < syspage_common.mapsCnt; i++, pmap++) {
+		if (pmap->map.id == id)
+			return pmap->map.name;
+	}
+
+	return NULL;
+}
+
```

Comment:
Output formatting when maps are displayed is the responsibility of Hubert.

Reply:
Ok, I'll remove that.

Reply:
I will ask Hubert to consider your changes as the better method of map presentation but the presentation is the responsibility of hardware independent part.

Reply:
Thank You.